projects/rocprofiler-systems/docs/how-to/openmp-profiling.rst

```diff
@@ -0,0 +1,115 @@
+    :keywords: rocprof-sys, rocprofiler-systems, ROCm, tips, how to, profiler, tracking, OpenMP, OMPT, AMD, Offload
+
+********************************************
+OpenMP performance profiling
+********************************************
+
+`ROCm Systems Profiler <https://github.com/ROCm/rocm-systems/tree/develop/projects/rocprofiler-systems>`_ supports profiling of OpenMP programs on ROCm 6.4.0 or higher.
+OpenMP performance data is captured by intercepting callbacks generated by the OpenMP Tools Interface (OMPT).
+
+Only a subset of the OMPT callbacks are processed:
+
+.. code-block:: shell
+
+  |----------------------------------+---------------------------|
+  |          OMPT Callback           |        Track Name         |
+  |----------------------------------+---------------------------|
+  | ompt_callback_cancel             | omp_cancel                |
+  | ompt_callback_dependences        | omp_dependences           |
+  | ompt_callback_device_finalize    | omp_device_finalize       |
+  | ompt_callback_device_initialize  | omp_device_initialize     |
+  | ompt_callback_device_load        | omp_device_load           |
+  | ompt_callback_dispatch           | omp_dispatch              |
+  | ompt_callback_error              | omp_error                 |
+  | ompt_callback_flush              | omp_flush                 |
+  | ompt_callback_implicit_task      | omp_implicit_task         |
+  | ompt_callback_lock_destroy       | omp_lock_destroy          |
+  | ompt_callback_lock_init          | omp_lock_init             |
+  | ompt_callback_masked             | omp_masked                |
+  | ompt_callback_mutex_acquire      | omp_mutex_acquire         |
+  | ompt_callback_mutex_acquired     | omp_mutex_acquired        |
+  | ompt_callback_mutex_released     | omp_mutex_released        |
+  | ompt_callback_nest_lock          | omp_nest_lock             |
+  | ompt_callback_parallel_begin     | omp_parallel              |
+  | ompt_callback_parallel_end       | omp_parallel              |
+  | ompt_callback_reduction          | omp_reduction             |
+  | ompt_callback_sync_region        | omp_sync_region           |
+  | ompt_callback_sync_region_wait   | omp_sync_region_wait      |
+  | ompt_callback_target_data_op_emi | omp_target_data_op_emi    |
+  | ompt_callback_target_emi         | omp_target_emi            |
+  | ompt_callback_target_submit_emi  | omp_target_submit_emi     |
+  | ompt_callback_task_create        | omp_task_create           |
+  | ompt_callback_task_dependence    | omp_task_dependence       |
+  | ompt_callback_task_schedule      | omp_task_schedule         |
+  | ompt_callback_work               | omp_work                  |
+  |----------------------------------+---------------------------|
+
+.. note::
+   ``omp_parallel`` tracks begin with ``ompt_callback_parallel_begin`` and ends when their corresponding ``ompt_callback_parallel_end`` is encountered.
+
+Configuration
```

Comment:
As indicated later in my review, we can move this section after showing instrument and run commands using `-I "ompt"` because they are much simpler. Make this configuration a sub section under "Collecting a trace with rocprof-sys-run". Similarly, you may want to show titles for different sections such as "Clone and Build the application", "Instrument the application with rocprof-sys-instrument", "Collecting a trace with rocprof-sys-run", and "Visualizing the trace with Perfetto"